File: src/study/_240312/junsu/최고의 집합.py
def solution(n, s):
    answer = []
    if n > s :
        answer.append(-1)
        return answer
    
    else:
        diff = s - s//n*n
        print(diff)
        for i in range(n-diff):
            answer.append(s//n)
        for i in range(diff):
            answer.append(s//n + 1)
        return answer

File: src/study/_240312/jeonghyeon/최고의집합.java
package study._240312.jeonghyeon;

class 최고의집합 {
    public int[] solution(int n, int s) {
        
        if(n > s)
            return new int[]{-1};
        
        int[] answer = new int[n];
        int basic = s/n;
        int additional = s%n;
        
        for(int i = 0; i < n-additional; i++)
            answer[i] = basic;
        for(int i = n-additional; i < n; i++)
            answer[i] = basic + 1;
        
        return answer;
    }
}